### DISCUSSION TITLE
About face codes

### DISCUSSION THREAD

**Discussion by yang1158280920** [exadel-inc/CompreFace#693] [Q&A]
Is there an API for obtaining facial feature codes and facial binary codes in the face API documentation?

**Comment by pospielov** [comment]
Not sure what "facial feature codes and facial binary codes" mean.
We have a "calculator" plugin that returns face embedding which is a vector of 512 or 128 numbers that represent the face. If it is what you want then check out this documentation:
https://github.com/exadel-inc/CompreFace/blob/master/docs/Face-services-and-plugins.md#face-plugins
